### PR TITLE
Add note about security descriptor

### PR DESCRIPTION
**Pull request by DesLee-msft**
Add note to let readers know that when the capture path is a subfolder instead of the root folder, the resulting image will inherit security descriptors from the folder.

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Minor style checks needed for this PR. Thanks.

**Comment by e0i** [comment]
Hello, @DesLee-msft . Please re-target the target branch of this PR to master so that we can proceed with the review. Thank you for your contribution.

**Comment by DesLee-msft** [comment]
@e0i , do I need to abandon this PR and create a new one to re-target to master? I'm not able to find the option to do that from the existing PR.

**Comment by e0i** [comment]
> @e0i , do I need to abandon this PR and create a new one to re-target to master? I'm not able to find the option to do that from the existing PR.

@DesLee-msft Hello, Please try clicking the branch link at the top of the PR where a menu should appear to choose between branches.

**Comment by DesLee-msft** [comment]
> @e0i , do I need to abandon this PR and create a new one to re-target to master? I'm not able to find the option to do that from the existing PR.
> 
> @DesLee-msft Hello, Please try clicking the branch link at the top of the PR where a menu should appear to choose between branches.

Clicking the branch link just took me to the page for that branch (https://github.com/MicrosoftDocs/windows-powershell-docs/tree/live)

**Comment by e0i** [comment]
> > @e0i , do I need to abandon this PR and create a new one to re-target to master? I'm not able to find the option to do that from the existing PR.
> > @DesLee-msft Hello, Please try clicking the branch link at the top of the PR where a menu should appear to choose between branches.
> 
> Clicking the branch link just took me to the page for that branch (https://github.com/MicrosoftDocs/windows-powershell-docs/tree/live)

Apologies, It should be available under after clicking *Edit* on the PR.

**Comment by DesLee-msft** [comment]
> @e0i , do I need to abandon this PR and create a new one to re-target to master? I'm not able to find the option to do that from the existing PR.
> @DesLee-msft Hello, Please try clicking the branch link at the top of the PR where a menu should appear to choose between branches.
> 
> Clicking the branch link just took me to the page for that branch (https://github.com/MicrosoftDocs/windows-powershell-docs/tree/live)
> 
> Apologies, It should be available under after clicking Edit on the PR.

That worked. Thanks!

File: docset/windows/dism/New-WindowsImage.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ This command captures the Drive D image in the WIM file located on d:\ and save
 ### -CapturePath
 Specifies the drive or path to the Windows operating system that is to be captured to an image file.
 
+   > [!Note]
```
Comment (JohanFreelancer9) [inline docset/windows/dism/New-WindowsImage.md]:
```suggestion
   > [!NOTE]
```

File: docset/windows/dism/New-WindowsImage.md [hunk 2]
```diff
@@ -54,6 +54,9 @@ This command captures the Drive D image in the WIM file located on d:\ and save
...
+   > When the path specified is not the root folder of a drive, the captured image will inherit the parent folder’s security descriptors. If you are capturing an image which was previously applied, apply the original image to the root folder of the drive to ensure that the security descriptors of the new image remain the same.
```
Comment (JohanFreelancer9) [inline docset/windows/dism/New-WindowsImage.md]:
```suggestion
   > When the path specified is not the root folder of a drive, the captured image will inherit the parent folder’s security descriptors. If you are capturing an image that was previously applied, apply the original image to the root folder of the drive to ensure that the security descriptors of the new image remain the same.
```

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by tseward** [comment]
Agree with @dariomws306 but otherwise the change looks good. For this part:

> parent folder’s security descriptors

It can just be:

> parent folder security descriptors

File: docset/windows/dism/New-WindowsImage.md [hunk 3]
```diff
@@ -54,6 +54,9 @@ This command captures the Drive D image in the WIM file located on d:\ and save
 ### -CapturePath
 Specifies the drive or path to the Windows operating system that is to be captured to an image file.
 
+   > [!NOTE]
```
Comment (JohanFreelancer9) [inline docset/windows/dism/New-WindowsImage.md]:
```suggestion
```

File: docset/windows/dism/New-WindowsImage.md [hunk 4]
```diff
@@ -54,6 +54,9 @@ This command captures the Drive D image in the WIM file located on d:\ and save
...
+   > When the path specified is not the root folder of a drive, the captured image will inherit the parent folder’s security descriptors. If you are capturing an image that was previously applied, apply the original image to the root folder of the drive to ensure that the security descriptors of the new image remain the same.
```
Comment (JohanFreelancer9) [inline docset/windows/dism/New-WindowsImage.md]:
```suggestion
When the path specified is not the root folder of a drive, the captured image will inherit the parent folder security descriptors. If you are capturing an image that was previously applied, apply the original image to the root folder of the drive to ensure that the security descriptors of the new image remain the same.
```

**Comment by e0i** [comment]
@DesLee-msft Apologies to bother again; Any chances to follow up with peer reviewer's revision suggestions above? Thank you for your time and patience with the process.

**Comment by DesLee-msft** [comment]
> @DesLee-msft Apologies to bother again; Any chances to follow up with peer reviewer's revision suggestions above? Thank you for your time and patience with the process.

Sorry. This fell off my radar for a few days. Will take a look today.

**Comment by DesLee-msft** [comment]
> @DesLee-msft Apologies to bother again; Any chances to follow up with peer reviewer's revision suggestions above? Thank you for your time and patience with the process.

Reviewed and accepted both suggestions.

**Comment by e0i** [comment]
@DesLee-msft Thank you. Passing to the PubOps for merge after all the checks have been completed.

**Pull request merged**